README.md

```diff
@@ -1,7 +1,7 @@
 # Calendar
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://Ksenia-Didukh.github.io/layout_calendar/)
```

Comment:
gh-pages не работает ссылка, ты сделала `npm run deploy`?

Reply:
да, делала,
$ npm run deploy

> calendar@1.0.0 deploy D:\projects\layout_calendar
> mate-scripts deploy

Start deploy to gh-pages. Please wait, it may take up to minute.

Warning: No html_report
Successfully deployed to gh-pages!